server/errors/errors.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package errors
+
+import (
+	te "github.com/google/trillian/errors"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+)
+
+// WrapError wraps err as a GRPC error if err is a TrillianError, else err is
+// returned unmodified.
+func WrapError(err error) error {
```

Comment:
The intent was:

1. Make trillian/errors "standalone" (no external dependencies)
1. Isolate the GRPC dependency **and** have the translation layer exclusively on the server package. This is statement that the translation is not seen as a responsibility of trillian/errors; if other packages want to translate it that's fine, but it's their responsibility.

There's also the TrillianError vs GRPC discussion, which is in a previous comment thread.